addons/source-python/packages/source-python/translations/strings.py

```diff
@@ -291,5 +303,20 @@ def get_language(self, language):
         # Return None as the language, as no language has been found
         return None
 
+    def tokenized(self, **tokens):
+        """Create a new TranslationStrings instance and store tokens in it.
+
+        :param dict tokens: Tokens to store in the instance.
+        :return: New TranslationStrings instance with tokens stored in it.
+        :rtype: TranslationStrings
+        """
+        result = TranslationStrings()
+        result.tokens.update(tokens)
+
```

Comment:
This could just be ``result.update(self)``. But we might even want to create a deep copy.